includes/wikia/GlobalTitle.php

```diff
@@ -713,12 +713,8 @@ private function loadServer() {
 	 * @return string
 	 */
 	private static function normalizeEnvURL( $server ) {
-		global $wgWikiaEnvironment;
-		if ( $wgWikiaEnvironment != WIKIA_ENV_PROD ) {
-			return WikiFactory::getLocalEnvURL( $server );
-		}
-
-		return $server;
+		// TODO MAIN-7994 use WikiFactory::getLocalEnvURL when it supports custom domains
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)